services/brig/test/integration/Federation/Util.hs

```diff
@@ -108,3 +113,23 @@ assertRight = \case
 
 assertRightT :: (MonadIO m, Show a, HasCallStack) => ExceptT a m b -> m b
 assertRightT = assertRight <=< runExceptT
+
+testQualifiedGetConversation :: Galley -> String -> User -> User -> ConvId -> Http ()
```

Comment:
This utility function is a bit confusing. Could you document what its arguments are? For example, it seems that `alice` is only used for its domain.